docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,36 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
+
+Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown. In most cases, we only want to invest the time in writing an ADR only when a decision has been made and needs to be codified. Until then, it's a Decision Proposal, not a Decision Record.
+
+Instead we use a Request for Comment markdown template that is more lightweight and speculative. It is designed to generate conversation and rapid feedback when there is not a clear solution with any level of certainty.
+
+RFCs live as PR templates in the ddev-live docs repo.
+
+An RFC template contains the following fields:
+* The Problem
+* Potential Solutions (if any)
+* Areas of Potential Impact (on the platform, company etc.)
+* How I Will Know I’ve Solved the Problem
+
+Workflow:
+* Open an RFC template PR in ddev-live with the 4 fields populated
+* Link the PR to the original source issue that led to the RFC
```

Comment:
Could we mark this step as **optional/recommended** in some way? I'd beg to say that this leads me to believe that an RFC can only be created if it has a relevant issue that it can be attached to, which I don't think should **always** be the case...

Reply:
I disagree. I don't think RFCs should happen in a vacuum; they should be related to planned work.  It may be an indirect relationship but making the linkage is still valuable.